extensions/applicationinsights-internplugin-js/uploadFilesToAzureStorage.ps1

```diff
@@ -0,0 +1,91 @@
+$storageAccountName = "daprodcdn";
+$cacheControlValue = "public, max-age=1800, immutable";
+$contentType = "text/javascript; charset=utf-8";
+
+function Show-Menu
```

Comment:
I don't believe you need this file unless you are planning to expose this in CDN

Reply:
We are planning on uploading this to the CDN, so that it's easier to include for different sites, even if they are using the snippet.